iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -439,7 +439,7 @@ def migrate_sections(self):
                     })
                     if row['description'] is None:
                         self.post_migration_report_messages['sections_with_null_description'].append(
-                            f'title: {translated_section.title}. URL: {translated_section.full_url}.')
+                            f'title: {translated_section.title}. URL: {translated_section.full_url}. Admin URL: http://localhost/admin/pages/{translated_section.id}/edit/')
```

Comment:
I think we should use the reverse URL function to generate this. That will also help make the domain dynamic.